LICENSE.rst

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2017, DESI Collaboration <desi-data@desi.lbl.gov>
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright notice, this
+  list of conditions and the following disclaimer in the documentation and/or
+  other materials provided with the distribution.
+* Neither the name of the copyright holder nor the names of its contributors
```

Comment:
As I've been updating license files, I've been using 'DESI Collaboration' instead of 'copyright holder.'  I recommend copying the file from the desitemplate product &adjusting the date on the first line as needed.